index.js

```diff
@@ -2,6 +2,13 @@
 const fs = require('fs');
 const path = require('path');
 
+class DuplicateCompatError extends Error {
+  constructor(message) {
+    super(message);
```

Comment:
Error text suggestions:

- Put the message text template in the constructor, so it's not possible to have inconsistent message text (should this ever be reused elsewhere)
- Avoid "please" in the case of operator error ([example style guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please))
- The count's not knowable here (there could be a third as-yet-unprocessed duplicate)

```suggestion
    super(`${feature} already exists! Remove duplicate entries.`);
```